static/templates/settings/image_upload_widget.hbs

```diff
@@ -1,5 +1,5 @@
 <div id="{{widget}}-upload-widget" class="inline-block image_upload_widget">
-    <div class="image-disabled {{#if is_editable_by_current_user}}hide{{/if}}">
+    <div class="image-disabled {{#unless is_editable_by_current_user}}hide{{/unless}}">
```

Comment:
This is not correct, we actually want to show this when changing image is disabled. This is currently used in the user profile avatar widget, where we show a text when changing avatar is disabled in organization.

Reply:
I have updated the PR.
So, is this the correct behaviour?
Or do we want some text to be shown to inform that the user doesn't have the permission to edit/delete the logo?

https://user-images.githubusercontent.com/85362194/161377912-ee9e4b0d-eee6-42b1-9941-4b0e388c1fe2.mov

Reply:
I think we would also want to avoid changing background where no text is visible for disabled state.

Reply:
Updated the PR by showing the changed background and and text only when the "disabled text" is available.